src/hardware/imf_rom.c

```diff
@@ -0,0 +1,1082 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Refer to "imf.cpp" for Copyright details.
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the Free
+ *  Software Foundation; either version 2 of the License, or (at your option)
+ *  any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ *  more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with
+ *  this program; if not, write to the Free Software Foundation, Inc., 51
+ *  Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+// Read-only memory tables for the IBM Music Feature
+
+// clang-format off
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Refer to "imfc.cpp" for Copyright details.
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the Free
+ *  Software Foundation; either version 2 of the License, or (at your option)
+ *  any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ *  more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with
+ *  this program; if not, write to the Free Software Foundation, Inc., 51
+ *  Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+// Read-only memory tables for the IBM Music Feature
+
+static constexpr uint8_t m_romPresetConfiguration16Binary[160] = {
```

Comment:
Get rid of the `m_` prefixes? Should be just `RomPresetConfiguration16Binary`, etc.

Reply:
@johnnovak Is that Hungarian notation?

Actual question, but it's funny to ask you that LOL

Reply:
Yeah, I think so... 😄 It's called like that because it was introduced by [Charles Simonyi](https://en.wikipedia.org/wiki/Charles_Simonyi), who's Hungarian.

Reply:
(marking resolved because not renaming variables)